front/app/containers/Admin/projects/project/phase/phaseParticipationConfig/utils/getOutput.ts

```diff
@@ -1,98 +0,0 @@
-import { omitBy, isNil } from 'lodash-es';
-
-import { State, IPhaseParticipationConfig } from '..';
-
-export default ({
```

Comment:
This was a somewhat magical mechanism to get the attributes needed for a participation method. It's now obsolete because the configurations take care of this.